CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 All notable changes to this project will be documented in this file, which follows the guidelines
 on [Keep a CHANGELOG](http://keepachangelog.com/). This project adheres to
 [Semantic Versioning](http://semver.org/).
+## [Un-released]
```

Comment:
This is a duplicate heading - these sub-heading should be merged into the section below